src/hooks/useTokenList.ts

```diff
@@ -3,21 +3,52 @@ import { TokenDetails } from 'types'
 import useSafeState from './useSafeState'
 import { getTokens, subscribeToTokenList } from 'services'
 import { EMPTY_ARRAY } from 'const'
+import stablecoins from 'data/stablecoinList'
 
 export interface UseTokenListParams {
   networkId?: number
   excludeDeprecated?: boolean
+  stablecoinList?: typeof stablecoins
 }
 
-export const useTokenList = ({ networkId, excludeDeprecated }: UseTokenListParams = {}): TokenDetails[] => {
+function getNetworkCorrectStableCoinList(
+  networkId: number | undefined,
+  stablecoinList: typeof stablecoins,
```

Comment:
we said this is not a stable coin anymore, note we have for example WETH.
It's a `quoteTokenPriorities` or something similar